src/autolabel/labeler.py

```diff
@@ -43,13 +43,13 @@ class LabelingAgent:
 
     def __init__(
         self,
-        config: Union[str, Dict],
```

Comment:
can we still accept these, at least until we update the docs and share the change with Discord? 

will be backwards incompatible otherwise and break all of our examples/docs